cfgov/jinja2/v1/owning-a-home/_templates/brand-footer.html

```diff
@@ -0,0 +1,85 @@
+
+{# ==========================================================================
+
+   render(links)
+
+   ==========================================================================
+
+   Description:
+
+   Renders all the brand footer content: 3 link sections and email section.
+
+   links: An array of three link objects, each containing:
+            - text (req): text of link section
+            - img: img for link section
+            - label: link label
+            - url: link url
+
+            Macro will display max of three link sections. If there are
+            more than three link objects, only the first three will
+            be shown. If no links are passed in, only the email signup
```

Comment:
Could you update this comment for clarity. It's not obvious that the email signup is shown regardless of the number of links.

Reply:
I left this as-is for now, I don't have a better explanation for what the code accomplishes.